non-free.md

```diff
@@ -218,6 +218,7 @@
 **[`^        back to top        ^`](##awesome-selfhosted---non-free-software)**
 
 - [Bamboo](https://www.atlassian.com/software/bamboo) - Continuous integration server. `⊘ Proprietary` `Java`
+- [Grai](https://www.grai.io) `⊘ Proprietary` - Grai is the open source data observability tool that catches bugs before they make it to prod. Using data lineage it understands how changes impact your data stack and reports back in version control. ([Source Code](https://www.github.com/grai-io/grai-core)) `Elastic-2.0` `Docker`
```

Comment:
- The description should explain what the software does
- If the software is actually under the Elastic-2.0 license, please remove the `⊘ Proprietary` label.

Reply:
@nodiscc

I've attempted to reword the description more to your liking but please let me know if you need any additional modifications.

> If the software is actually under the Elastic-2.0 license, please remove the ⊘ Proprietary label.

Apologies, this followed from the description build utility. I had assumed it was a required label in the non-free.